build/scripts/test-determinism.ps1

```diff
@@ -191,6 +191,10 @@ function Run-Test() {
     $altRootDir = Join-Path "$repoDir\Binaries" "q"
     Remove-Item -re -fo $altRootDir -ErrorAction SilentlyContinue
     & robocopy $repoDir $altRootDir /E /XD $binariesDir /XD ".git" /njh /njs /ndl /nc /ns /np /nfl
+
+    # Symlink the .git directory to make SourceLink think Binaries/q is the repo root:
+    cmd /c mklink /d (Join-Path $altRootDir ".git") (Join-Path $repoDir ".git")
```

Comment:
Please prefix this with &

Reply:
It causes powershell to parse the command line in a more windows command friendly way. I've forgotten the exact details frankly but my mental model is "when running a windows command use &". 

---
In reply to: [196531824](https://github.com/dotnet/roslyn/pull/26981#discussion_r196531824) [](ancestors = 196531824)